internal/project/framework/internals/middleware/server_stats.go

```diff
@@ -8,8 +8,9 @@ import (
 	"sync"
 	"time"
 
-	ierror "{{ .PkgPath }}/framework/internals/error"
-	"{{ .PkgPath }}/framework/internals/latency"
+	/**#bean*/
+	ierror "demo/framework/internals/error" /*#bean.replace(ierror "{{ .PkgPath }}/framework/internals/error")**/
```

Comment:
put these two lines in one line

Reply:
`go fmt` command make it, `go fmt` will run before I commit,but I don't know the fmt rules
bean macro still works in two lines